### PR TITLE
Tolerate thrift IDL annotations

### PR DESCRIPTION
**Pull request by kriskowal**

### CODE REVIEW

**Comment by kriskowal** [comment]
r @malandrew @leizha @Raynos

**Comment by malandrew** [comment]
LGTM. The only comment is if there are a lot more places in which type annotations are relevant. Is it only structs and fields or are type annotations supported on a lot more types. IIRC correctly the answer is yes.

File: test/annotations.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+// THE SOFTWARE.
+
+
+'use strict';
+
+var path = require('path');
+var thriftify = require('../');
+var tape = require('tape');
+
+tape('tolerate annotations', function t(assert) {
```
Comment (leizha) [inline test/annotations.js]: verify the type annotation?

**Comment by leizha** [comment]
LGTM

**Comment by kriskowal** [comment]
@malandrew Let’s revisit and make sure that we’ve covered all the annotatable terms in another pass. Also: in a future pass, thread annotations through.

**Pull request merged**